modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3053,7 +3056,7 @@ else if (selectExpression instanceof ScalarDSLQueryAlias) {
 
 	private void _insertIntoLocalizationTable(
 			ObjectDefinition objectDefinition, long objectEntryId,
-			Map<String, Serializable> values)
+			ServiceContext serviceContext, Map<String, Serializable> values)
```

Comment:
What do you thing to pass only `int workflowAction` in all private methods this class?